.github/workflows/docker.yml

```diff
@@ -23,6 +23,10 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Decrypt service-account.json
+        run: ./spec/quarantine/decrypt_secret.sh
+        env:
+          QUARANTINE_SERVICE_ACCOUNT_PASSPHRASE: ${{ secrets.QUARANTINE_SERVICE_ACCOUNT_PASSPHRASE }}
```

Comment:
I am doing something similar. Unfortunately, this does not work on PRs are pushed from  a fork because secrets do not get passed to runner pushed from a fork (https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow). 

Ex: I pulled down this PR and pushed it myself and it failed https://github.com/rubyforgood/casa/actions/runs/8888424582/job/24405217184

I just ran into this issue so I don't have a solution unfortunately.

Reply:
It's failing now cause the secret doesn't exist. It could also be the fork issue.